sites/dotnet-guide/contents/tutorials/basics/ado-net/index.md

```diff
@@ -0,0 +1,104 @@
+---
+type: TutorialStep
+date: 2020-06-05
+title: Basics of ADO.NET
+technologies: [.net]
+products: [rider]
+topics: [data]
+author: khalidabuhakmeh
+subtitle: Working with databases using DbConnection, DbCommand, and DbDataReader
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+#   url: TODO: Upload video    
+---
+
+ADO.NET is a data service for .NET developers. In the older version of the .NET framework, Microsoft packaged it along with other libraries. More recently, the ADO.NET libraries ship in NuGet packages, with `System.Data.SqlClient` being specifically for SQL Server access. ADO.NET is a low-level interface to our data storage of choice, and its use depends heavily on our understanding of SQL.
+
+ADO.NET has three commonly used objects that developers will deal with: `DbConnection`, `DbCommand`, and `DbDataReader`. The SQL Server implementations of these classes are `SqlConnection`, `SqlCommand`, and `SqlDataReader`. 
+
+ADO.NET code always starts with an open database connection. The class allows us to pass commands and read results from a database. We can perform operations through the `DbCommand` implementation. With `SqlCommand`, we can run text, stored procedures, and table direct queries. When executing a SQL query, we can read any results via a `DbDataReader`. Mapping the results to our C# models is our responsibility, and can be tedious.
+
+ADO.NET assumes very little for us, and it is the lowest level approach to data access available to .NET developers. For straightforward scenarios, ADO.NET is a capable and enjoyable method for data access. We may want to consider other approaches when our use cases become more involved or find reusable patterns emerging in our code base.
+
+### Code Snippets
```

Comment:
Random question: would it make sense to share the project somewhere? Makes maintenance easier if needed? (plus folks can clone and run)